README.md

```diff
@@ -130,6 +130,8 @@ This file must be located in the project root and declares the project settings.
 | assets                    |                             | static asset files, that should be copied to the styleguide directory, head for "Assets" for further information
 | content                   |                             | list of page configurations, take a look at "Content"
 | componentDocs             | "."                         | path of component doc files, relative to the given component.yml. "doc" e.g says "all documents have to be placed in a sub-directory doc adjacent to the component.yml"
+| assetHost                 |                             | url origin for external assets
+| themes                    |                             | list of style themes, of that each defines a different themed version of the components
```

Comment:
no thats perfectly ok to me. Inside the layout context it is ok, because the main "theme" property is irrelevant for it. For me it is only a  matter for the public API because lacking an appropriate plugin solution for things like that.